packages/cozy-stack-client/src/DocumentCollection.js

```diff
@@ -316,14 +317,44 @@ class DocumentCollection {
   /**
    * Use Couch _changes API
    *
-   * @param  {String} since     Starting sequence for changes
-   * @param  {Object} options   { includeDesign: false, includeDeleted: false }
+   * @param  {Object} couchOptions Couch options for changes https://kutt.it/5r7MNQ
+   * @param  {Object} options      { includeDesign: false, includeDeleted: false }
    */
-  async fetchChanges(since, options = {}) {
-    const result = await this.stackClient.fetchJSON(
-      'GET',
-      `/data/${this.doctype}/_changes?include_docs=true&since=${since}`
-    )
+  async fetchChanges(couchOptions = {}, options = {}) {
+    const haveDocsIds = couchOptions.doc_ids && couchOptions.doc_ids.length > 0
```

Comment:
`hasDocIds`

Can we have `docIds` instead of `doc_ids` ? This will be more javascript-y

Reply:
not really `doc_ids` is couch parameter, [see documentation](https://docs.couchdb.org/en/2.3.0/api/database/changes.html) ;)